src/includes/closures/Plugin/InstallUtils.php

```diff
@@ -200,6 +204,108 @@
 };
 
 /*
+ * Add template blocks to `/.htaccess` file.
+ *
+ * @since 15xxxx Adding `.htaccess` tweaks.
+ *
+ * @return boolean True if added successfully.
+ */
+$self->addWpHtaccess = function () use ($self) {
+    if (!$self->options['enable']) {
+        return false; // Nothing to do.
+    }
+    if (!$self->removeWpHtaccess()) {
+        return false; // Unable to remove.
+    }
+    if (!($htaccess_file = $self->findHtaccessFile())) {
+        if (!is_writable(ABSPATH) || file_put_contents($htaccess_file = ABSPATH.'.htaccess', '') === false) {
+            return false; // Unable to find and/or create `.htaccess`.
+        } // If it doesn't exist, we create the `.htaccess` file here.
+    }
+    if (!is_readable($htaccess_file)) {
+        return false; // Not possible.
+    }
+    if (($htaccess_file_contents = file_get_contents($htaccess_file)) === false) {
+        return false; // Failure; could not read file.
+    }
+    $template_blocks = '# BEGIN '.GLOBAL_NS."\n"; // Initialize.
```

Comment:
@jaswsinc I noticed that we're using `GLOBAL_NS` here, instead of `NAME`, which seems more appropriate. (`# BEGIN ZenCache` looks nicer than `# BEGIN zencache`.) Is there any reason we're not using `NAME` here?